### ISSUE TITLE
Warnings emitted when installing dependencies

### ISSUE THREAD

**Issue by mcmire** [KeystoneHQ/ur-registry#15]
I tried to clone this repo and install dependencies, but in doing so I ran into a couple of things:

1. There seem to be both `yarn.lock` and `package-lock.json` present in the repo. Which package manager is preferred for development purposes? Keep in mind that Yarn will emit a warning if `package-lock.json` is present.
2. When using `npm install`, NPM (as of the latest version, v8) reports that `package-lock.json` was created with an older version of NPM. If NPM is preferred for installing dependencies, which version is preferred?

**Comment by aaronisme** [comment]
good catch, we are using the yarn instead of npm. we will remove the pacakge-lock.json
